micro_dl/inference/evaluation_metrics.py

```diff
@@ -165,6 +319,18 @@ def binarize_array(im):
     return im_bin.astype(np.uint8)
 
 
+def cpmask_array(im, model_name):
+    """mask preparation using cellpose
+
+    :input model: cellpose model used for segmentation
+    :input im: predicted image to be segmented
+    :return im_mask: cellpose segmented mask
+    """
+    model = models.CellposeModel(model_type=model_name)
+    channels = [0, 0]
+    masks = model.eval(im, channels=channels, diameter=None)[0]
```

Comment:
Nothing is returned here.

Reply:
Thanks for pointing out! I have fixed it.